src/components/Person/Person.jsx

```diff
@@ -1 +1,45 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import PropTypes from 'prop-types';
+import './Person.scss';
+
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  const partner = sex === 'm'
+    ? 'wife'
+    : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age &&
+        <p className="Person__age">{`I am ${age}`}</p>
+      }
```

Comment:
you probably shouldn't commit suggestions through github UI
this will give you an error in VSCode :)